README.md

```diff
@@ -59,6 +59,22 @@ This option `default` is where you may specify which of the connections below yo
 
 This option `connections` is where each of the connections are setup for your application. Example configuration has been included, but you may add as many connections as you would like.
 
+#### Encrypted Channels
+To enable (end to end encrypted channels)[https://pusher.com/docs/client_api_guide/client_encrypted_channels], you need to uncomment a line from the Pusher config file
+
+```
+'app_id' => env('PUSHER_APP_ID'),
+'options' => [
+    'cluster' => env('PUSHER_APP_CLUSTER'),
+    'encryption_master_key' => env('PUSHER_ENCRYPTION_MASTER_KEY'),
+],
+'host' => null,
+'port' => null,
+```
+
+Then you need to set an `encryption_master_key` in your `.env` file. You should then be able to publish encrypted events to channels prefixed with `private-encrypted` and you can validate this is working by checking the (dashboard)[https://dashboard.pusher.com] debug console for your app!
```

Comment:
@kn100 think your markdown link might be broken - the brackets are usually the other way round